packages/components/src/checkbox-control/style.scss

```diff
@@ -84,9 +84,24 @@ svg.components-checkbox-control__indeterminate {
 	@include break-small() {
 		--checkmark-size: calc(var(--checkbox-input-size) + 4px);
 	}
+
+
+	animation: components-checkbox__appear-animation 0.1s ease-in-out;
```

Comment:
If we wanted to show an animation also when unchecking the checkbox, we could use `transition` instead of animation, and apply the two different transition states using on `input:checked + icon` selector.

Also, what's the deal with the indeterminate state? Should we animate to/from it?